package.json

```diff
@@ -23,28 +26,33 @@
     "url": "https://github.com/imagekit-developer/imagekit-vuejs"
   },
   "dependencies": {
-    "imagekit-javascript": "^1.3.5"
+    "babel-eslint": "^10.1.0",
+    "imagekit-javascript": "^2.0.0",
+    "vue": "^3.2.47"
```

Comment:
vue should be a peer dependency because people already have a vue.js app and installing this SDK in that project. Installing this SDK should not install separate copy of vue.js. We do the same in react.js SDK. After this is fixed, please update the package-lock.json as well.

Reply:
Done

Reply:
Why is `vue` still in dependency list?

<img width="1087" alt="image" src="https://github.com/imagekit-developer/imagekit-vuejs/assets/45416977/91bd4271-8cb8-4e58-b098-74d52d78ee86">